### PR TITLE
Add user research banner

### PR DESCRIPTION
**Pull request by hannako**
As requested by MOD via our normal processes
https://trello.com/c/2RgPDQf8/2800-govuk-user-research-banner-requests-mod

|Before|After|
|-|-|
|Screenshot 2024-08-20 at 12 52 38|Screenshot 2024-08-20 at 12 52 06|

Trello: https://trello.com/c/2RgPDQf8/2800-govuk-user-research-banner-requests-mod
Review app: https://government-frontend-pr-3304.herokuapp.com/guidance/medals-campaigns-descriptions-and-eligibility

### CODE REVIEW

File: app/views/content_items/detailed_guide.html.erb [hunk 1]
```diff
@@ -3,6 +3,7 @@
     schema: :faq
   ) %>
 <% end %>
+<%= render 'shared/intervention_banner' %>
```
Comment (leenagupte) [inline app/views/content_items/detailed_guide.html.erb:6]: Should this be in a guard clause? So it doesn't need to be added and removed everytime?
e.g. https://github.com/alphagov/frontend/blob/735e8c9e1d5bcab90994651009437e9127e37b58/app/views/shared/_intervention_banner.html.erb#L1

Comment (leenagupte) [reply app/views/content_items/detailed_guide.html.erb:6]: I think the field values should be added too?

**Review by KludgeKML — APPROVED** [review]
LGTM

**Review by leenagupte — COMMENTED** [review]
A couple of inline comments.

File: app/views/content_items/detailed_guide.html.erb (hunk 1 above)
Comment (leenagupte) [reply app/views/content_items/detailed_guide.html.erb:6]: Sorry, I missed that it's in it's own partial 🤦‍♀️

**Pull request merged**